docs/data/material/guides/styled-engine/styled-engine.md

```diff
@@ -113,3 +113,75 @@ You can use these `styled-component` examples as a reference:
 :::
 
 This package-swap approach is identical to the replacement of React with [Preact](https://github.com/preactjs/preact). The Preact team has documented a large number of installation configurations. If you are stuck with Material UI + styled-components, don't hesitate to check out how they solve the problem, as you can likely transfer the solution.
+
```

Comment:
Can we add a general section for how it works with other UI libraries using emotion, and then have these two examples for Theme UI and Chakra UI?

Reply:
Done!